multiview/utils/cluster_utils.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2019 NeuroData (http://neurodata.io)
+#
```

Comment:
Is this file needed? Can we put these utils into the utils.py file?

Reply:
I think the difference in his utils is that the need for two views is enforced? @achang63 lmk if this isn't true.

What might be better is to remove this file and just have that additional check be part of Kmeans? It's also fine to have your own utils file for later functions but simplify it so that there isn't code repetition between utils/utils.py and this file.

Reply:
If you need 2 views enforced, that can be combined with the new utils, because the CTClassifier commit added arguments to the Check_Xs util to enforce a certain number of views